Dockerfile

```diff
@@ -0,0 +1,17 @@
+# Copyright 2021 Gravitational, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+FROM gcr.io/distroless/base
+EXPOSE 80
```

Comment:
As per Kevin's comment in the Helm chart, maybe we should think about making this listen on a higher port (like 3000) so we can run as a non-root user?

Reply:
Changed in 6f2087b to 3888.